book-a-desk-ui/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Looks like we have double `package.json` now?
https://github.com/book-a-desk/book-a-desk-ui/blob/1b9a7ba276428a8d2a7fbf841bc476dbe880b370/package.json
https://github.com/book-a-desk/book-a-desk-ui/blob/1b9a7ba276428a8d2a7fbf841bc476dbe880b370/book-a-desk-ui/package.json